src/controller.cpp

```diff
@@ -1059,3 +1059,15 @@ bool Controller::deactivate() {
   m_impl->deactivate(stateToReason(m_state));
   return true;
 }
+
+void Controller::forceDaemonSilentServerSwitch() {
+#ifdef MZ_IOS
+  if (m_impl) {
```

Comment:
Consider logging errors if m_impl is not present.

Reply:
Added, thanks